### DISCUSSION TITLE
AETNA missing Insurer MRFs

### DISCUSSION THREAD

**Discussion by hw-monkey** [CMSgov/price-transparency-guide#624] [Q&A]
When looking at:
https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICSI/latest_metadata.json

I am only seeing "Third Party Administrator" and "Rental Network" files. There are not any "Insurer" files. Similarly, if I use their interface to search by Insurer or Issuer, there are not any legitimate results that would reflect the plans they self-manage.

Am I missing something, or is AETNA?

**Comment by Ginger228** [comment]
Hi- Did you go to the Aetna.com?  They do have multiple files health insurance insurer.

**Reply by hw-monkey** [reply]
@Ginger228 
I truly do not understand the recommendation.  Can you provide a link to one MRF that is not third party or rental network, and how you found that MRF? Or, when using the aetna.com interface for searching ( https://health1.aetna.com/app/public/#/one/insurerCode=AETNACVS_I&brandCode=ALICSI/machine-readable-transparency-in-coverage ), what search term did you use to locate a non-third-party, non-rental-network, Insurer MRFs? As far as I can tell (after using an automated method to scan every result provided by that search tool), there isn't a single INSURER file. My automated method might be flawed, for it uses the first URL that I posted, which others have suggested contains links to all the MRFs.

From your other posts, I am assuming you work for AETNA. Can you please ask internally?

**Reply by Ginger228** [reply]
@hw-monkey  Aetna published external communication that the fully insured plans would be posted on the Aetna.com site.  When I searched the site the data was posted. 

Aetna.com- scroll to the bottom for Plan Disclosures- scroll until you come to Transparency in Coverage.  There are links to Aetna Student Health, Fully Insured Plans, Individual ACA Plans, & Small Group AFA plans.

**Reply by jeffld** [reply]
It looks like Aetna is forcing the use of a web browser interface to download the file.  Why hide the the url to the gzip file? Are they trying to make it inconvenient to automate the pull of the file?
